metashare/api/models.py

```diff
@@ -218,6 +222,29 @@ class Meta:
         unique_together = (("name", "project"),)
 
 
+SCRATCH_ORG_TYPES = Choices("Dev", "QA")
+
+
+class ScratchOrg(mixins.HashIdMixin, mixins.TimestampsMixin, models.Model):
+    task = models.ForeignKey(Task, on_delete=models.PROTECT)
+    org_type = StringField(choices=SCRATCH_ORG_TYPES)
+    owner = models.ForeignKey(User, on_delete=models.PROTECT)
+    last_modified_at = models.DateTimeField(null=True)
+    expires_at = models.DateTimeField()
+    latest_commit = StringField(blank=True)
+    latest_commit_url = models.URLField(blank=True)
+    url = models.URLField()
+    has_changes = models.BooleanField(default=False)
```

Comment:
@wlonk For create, can we make `owner`, `expires_at`, and `url` optional?

Reply:
Yeah; `owner` should actually get set automatically, too.